setup-ovs.sh

```diff
@@ -67,7 +75,17 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
       nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
   else
       nmcli conn mod brcnv-iface connection.autoconnect yes
+      nmcli conn up ovs-slave-$secondary_device
```

Comment:
if we agree to use consistent slave connection names, this would change to 
```suggestion
      nmcli conn up ovs-slave-2
```

Reply:
This is a fix that was done because we had an issue with the second interface. 
If we change the names then I am not sure that will stay

Reply:
It should work if we set `conn.id` when attaching links to the bonding.

Reply:
I think it's con-name. no? (it works [here](https://github.com/coreos/coreos-assembler/pull/2367/files#diff-d3d2a708b2f5727515ca091586b987a845a198307fc62584ce5047654ddfdfafR380))

Reply:
I used conn.id {Name}